src/main/java/com/lorepo/icplayer/client/module/text/TextView.java

```diff
@@ -21,9 +21,11 @@
 import com.lorepo.icplayer.client.module.IWCAGModuleView;
 import com.lorepo.icplayer.client.module.text.TextPresenter.IDisplay;
 import com.lorepo.icplayer.client.module.text.TextPresenter.TextElementDisplay;
+import com.lorepo.icplayer.client.module.text.TextPresenter.NavigationTextElement;
 import com.lorepo.icplayer.client.page.PageController;
 import com.lorepo.icplayer.client.utils.MathJax;
 import com.lorepo.icplayer.client.utils.MathJaxElement;
+import com.lorepo.icf.utils.JavaScriptUtils;
```

Comment:
Utilsy do czegoś się przydają czy tylko do logowania?

Reply:
BUMP - btw. dwa importy JavaScriptUtils w tym pliku - linijka 11 i 28. Dodatkowo linijka 837 jest log jakiś (nie Twoje zmiany). Może można go też wyoutować? Jest to raczej do debugowania aniżeli informacyjny jak np. przy errorze.